daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1141,55 +1136,50 @@ private[lf] object SBuiltin {
     final protected def importCid(cid: V.ContractId): SEImportValue =
       SEImportValue(typ, V.ValueContractId(cid))
     // Callback from the engine returned NotFound
-    def handleInputKeyNotFound(machine: Machine): Boolean
-    // CallBack from the engine returned a new Cid
-    def handleInputKeyFound(machine: Machine, cid: V.ContractId): Unit
+    def handleInputKeyFound(cid: V.ContractId): SExpr
```

Comment:
I’m confused why are there suddenly so many changes around interpretiaon in a PR that was about changing preprocessing errors?

Reply:
A commit sneaked in. I remove it.